src/pages/bridge/Transfers.tsx

```diff
@@ -53,14 +53,17 @@ function Transfers(): JSX.Element {
       const entries: TTransfer[] = [];
 
       issueEntries.forEach((e) => {
+        if (!walletAccount || e.request.requester.eq(walletAccount?.address)) {
```

Comment:
The condition is wrong and will only show the requests that the account did not create.

```suggestion
        if (!walletAccount || !e.request.requester.eq(walletAccount?.address)) {
```